misk-hibernate/src/main/kotlin/misk/hibernate/SchemaMigrator.kt

```diff
@@ -152,22 +154,35 @@ internal class SchemaMigrator(
       } catch (e: PersistenceException) {
         transacter.get().transaction(shard) { s ->
           s.useConnection { c ->
-            val statement = c.createStatement()
-            statement.addBatch("""
-            |CREATE TABLE schema_version (
-            |  version varchar(50) NOT NULL,
-            |  installed_by varchar(30) DEFAULT NULL,
-            |  UNIQUE KEY (version)
-            |);
-            |""".trimMargin())
-            statement.executeBatch()
+            createSchemaVersion(c)
           }
           sortedSetOf<NamedspacedMigration>()
         }
       }
     }
   }
 
+  private fun createSchemaVersion(c: Connection) {
+    val statement = c.createStatement()
+    if (transacter.get().config().type == DataSourceType.COCKROACHDB) {
+      statement.addBatch("""
```

Comment:
I think both support `version VARCHAR(50) UNIQUE NOT NULL`.

Reply:
`version VARCHAR(50) PRIMARY KEY` even!